src/complex/DataStructure/Geometry/TetrahedralGeom.hpp

```diff
@@ -15,6 +15,8 @@ class COMPLEX_EXPORT TetrahedralGeom : public AbstractGeometry3D
 public:
   friend class DataStructure;
 
+  static const int k_NumVerts = 4;
```

Comment:
```suggestion
  static inline constexpr usize k_NumVerts = 4;
```
I don't think these are supposed to be different data types.